geschaeftsordnung-vollversammlung.md

```diff
@@ -55,7 +55,7 @@ Anträge können vor der Behandlung durch die antragstellende Person zurückgezo
 
 (2) Anträge zur Geschäftsordnung werden vor Änderungsanträgen behandelt.
 
-(3) Angenommene Geschäftsordnungsanträge stehen über den Entscheidungen der Versammlungsleitung.
+(3) Angenommene Geschäftsordnungsanträge stehen über den Bestimmungen dieser Geschäftsordnung.
```

Comment:
Warum diese Änderung? KLingt in meinen Ohren irgendwie komisch. So als würde die Geschäftsordnung durch GO-Anträge entwertet werden.

Reply:
+1

Reply:
go-anträge können die go nicht ändern. es muss also explizit in der go ausformuliert sein, wie nach einem angenommenen go-antrag verfahren wird. der go-antrag ist dann nur der trigger, dass etwas bestimmtes in der go definiertes getan wird.

Reply:
@janno42 Kannst du bitte zumindest ein Beispiel geben, wie das konkret formuliert aussieht?